regression/contracts/assigns-replace-ignored-return-value/test.desc

```diff
@@ -1,11 +1,13 @@
 CORE
 main.c
 --replace-call-with-contract bar --replace-call-with-contract baz --enforce-contract foo
+^\[bar.precondition.\d+\] line \d+ Check bar's requires clause in foo: SUCCESS$
+^\[baz.precondition.\d+\] line \d+ Check baz's requires clause in foo: SUCCESS$
 ^EXIT=0$
 ^SIGNAL=0$
 ^VERIFICATION SUCCESSFUL$
 --
-^\[.*assigns.*\].*ignored_return_value.*FAILURE
+^\\[.*assigns.*\\].*ignored_return_value.*FAILURE
```

Comment:
Why are you escaping the backslashes? I'm pretty sure this pattern will now not match any output, but will spuriously be doing this, and therefore make the test not meaningful anymore.

Reply:
This was a typo. Removed it.